doc/builders/special/mkshell.section.md

```diff
@@ -15,3 +15,41 @@ pkgs.mkShell {
   inputsFrom = with pkgs; [ hello gnutar ];
 }
 ```
+# pkgs.mkShellMinimal {#sec-pkgs-mkShellMinimal}
```

Comment:
@Mic92 @roberth added some docs.
(I'm sure this is going to spur a lot of edits :smile:)

Reply:
I added it to the same file for now :shrug:

Reply:
Separate file now!